README.md

```
@@ -1,3 +1,50 @@
+- [x] 보드에서 기물의 위치를 바꾼다.
+
+### 입출력
+
+- [x] 장기판을 출력한다.
+- [x] 사용자로부터 좌표를 입력받고 이동한다.
+
+### 고려해야할 것
+
+- ✅ 장기의 말(駒)들은 어떻게 객체로 나누어야 하는가? && ✅ 각 말의 이동 규칙을 어떻게 설계할 것인가?
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)